packages/react/src/button/Button.css.ts

```diff
@@ -69,51 +69,51 @@ export const button = recipe({
       danger: {
         "@layer": {
           [layers.axiom]: {
-            vars: {
-              [accentColorVar]: theme.colors["bg.error.solid"],
-              [solidAccentColorVar]: theme.colors["bg.error.solid.hover"],
-              [subtleAccentColorVar]: theme.colors["bg.error.subtle"],
-            },
-
             selectors: {
               "&:focus-visible": {
                 outlineColor: "red.200",
               },
             },
+
+            vars: {
+              [accentColorVar]: theme.colors["bg.error.solid"],
+              [solidAccentColorVar]: theme.colors["bg.error.solid.hover"],
+              [subtleAccentColorVar]: theme.colors["bg.error.subtle"],
+            },
           },
         },
       },
       primary: {
         "@layer": {
           [layers.axiom]: {
-            vars: {
-              [accentColorVar]: theme.colors["bg.brand.solid"],
-              [solidAccentColorVar]: theme.colors["bg.brand.solid.hover"],
-              [subtleAccentColorVar]: theme.colors["bg.brand.subtle"],
-            },
-
```

Comment:
Can we revert these unnecessary changes?

Reply:
addressed bhai